### DISCUSSION TITLE
Calendar side-menu option missing

### DISCUSSION THREAD

**Discussion by elapse2039** [photoprism/photoprism#3150] [General]
I dont know exactly when this disappeared but perhaps in the most recent docker upgrade of my installation.  But the calendar menu tab on the right-hand panel is missing.  It was a very useful feature.

The version running currently is Build 221118-e58fee0fb

Any hints/tips on how to troubleshoot this?  I have not changed any Settings, usually just pull a new image and restart it.

**Comment by lastzero** [comment]
You mean in the mobile navigation menu? Was it possibly disabled in Settings?

**Reply by elapse2039** [reply]
What is the setting called that determines whether Calendar is present? I cant see one that jumps out at me. I have a few settings disabled, but always have. I have made no changes knowingly before noticing that mobile menu option disappearing.

**Reply by lastzero** [reply]
Moments, you can also simply try on our demo

**Reply by elapse2039** [reply]
Thx. Its never been enabled on my installation. I will enable it.

**Reply by Leon99** [reply]
I don't see Calendar in the mobile nav menu even when Moments is enabled. Even in the demo instance. It's visible in the full menu. 
![image](https://github.com/photoprism/photoprism/assets/6840381/e6114a94-5589-433b-91f9-da4d68085bef)

![image](https://github.com/photoprism/photoprism/assets/6840381/57874c9a-1db1-4024-bfc3-3f6434ef6e62)
@elapse2039 did you manage to get it?

**Reply by graciousgrey** [reply]
The menu on your second screenshot is not meant to show all options. To open the menu on the left side on your phone, you can simply click on the logo in the upper left corner.

**Reply by Leon99** [reply]
Oh I see. A bit confusing, but I can live with it. Thanks @graciousgrey!

**Comment by elapse2039** [comment]
![D84A035B-A613-4D32-96B6-F244A51CFFB5](https://user-images.githubusercontent.com/5325698/215336240-6ab83131-db42-4759-a258-f6de0a11db81.jpeg)

![15729BA9-53EB-4532-BAA4-E8471DCF7D43](https://user-images.githubusercontent.com/5325698/215336248-dbbbafb1-3dc1-4cee-9010-ceaaf96f201f.jpeg)

everything but the first two options are selected.
